Description: Add versioned soname to library
 Upstream has no versioned soname yet, so this patch adds a Debian
 specific soname version until something is added upstream.
Author: Andreas Bombe <aeb@debian.org>
Last-Update: 2016-12-11
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: git/makefile.in
===================================================================
--- makefile.in.orig
+++ makefile.in
@@ -1196,7 +1196,7 @@
 	${AR} r $@ $^
 	${RANLIB} $@
 
-libliquid.so: libliquid.a
+$(SHARED_LIB): libliquid.a
 	$(CC) $(CFLAGS) $(LDFLAGS) -shared -Xlinker -soname=$@ -o $@ -Wl,-whole-archive $^ -Wl,-no-whole-archive $(LIBS)
 
 # static archive and library objects
@@ -1221,6 +1221,7 @@
 	mkdir -p $(DESTDIR)$(exec_prefix)$(libdir)
 	mkdir -p $(DESTDIR)$(prefix)/include/liquid
 	install -m 644 -p ${ARCHIVE_LIB} ${SHARED_LIB} $(DESTDIR)$(exec_prefix)$(libdir)
+	ln -s $(SHARED_LIB) $(DESTDIR)$(exec_prefix)$(libdir)/libliquid.so
 	install -m 644 -p $(addprefix include/,$(headers_install)) $(DESTDIR)$(prefix)/include/liquid
 	@echo ""
 	@echo "---------------------------------------------------------"
--- configure.ac.orig
+++ configure.ac
@@ -239,7 +239,7 @@
     AC_PROG_AR
 
     AR_LIB=libliquid.a
-    SH_LIB=libliquid.so
+    SH_LIB=libliquid.so.1
     REBIND=ldconfig
     ;;
 esac
